static/scripts/dataprivacy/user-registration.js

```diff
@@ -36,6 +48,9 @@ window.addEventListener('DOMContentLoaded', () => {
 		});
 	}
 
+	if (passwordInput) passwordInput.onchange = validatePassword;
+	if (passwordConfirmInput) passwordConfirmInput.onkeyup = validatePassword;
```

Comment:
What was the reason to go with onchange, and onkeyup here? Is oninput an option in both cases?